docs/en/04_Changelogs/4.9.0.md

```diff
@@ -12,22 +12,34 @@
 
 ### Image lazy loading {#image-lazy-loading}
 
-Silverstripe CMS now lazy loads most images by default and includes some options to opt-out of this behaviour.
+Most modern browsers support the ability to "lazy load" images. When an image is configured to be 
+lazy loaded, browsers will only request the image once it's about to be visible to the users. This 
+reduces the initial rendering time for pages. 
 
-Image lazy loading defers the loading of images not in the viewport to improve the initial page load performance. 
-Read more about (image lazy-loading)[https://web.dev/browser-level-image-lazy-loading/].
+Silverstripe CMS 4.9 lazy loads most images by default. You have the options to opt-out of this 
+behaviour globally or on specific image instances.
 
 This feature was implemented in partnership with Google.
 
+Read [Browser-level image lazy-loading for the web](https://web.dev/browser-level-image-lazy-loading/)
+on _web.dev_ more information.
```

Comment:
This intro is for decision makers that don't care too much about the technical details.

Reply:
From v4.9 Silverstripe CMS lazy loads...